api/Highlight.json

```diff
@@ -89,7 +89,9 @@
             "chrome_android": "mirror",
             "edge": "mirror",
             "firefox": {
-              "version_added": "140"
+              "version_added": "140",
+              "partial_implementation": true,
+              "notes": "Multiple `add()` methods can not be chained, the must be added individually. See [bug 1969004](https://bugzil.la/1969004)."
```

Comment:
```suggestion
              "notes": "Does not return any value. See [bug 1969004](https://bugzil.la/1969004)."
```

Reply:
I feel that the more verbose note is needed.
This does not work:
```js
highlight.add(range1).add(range2);
```
However this does work:
```js
highlight.add(range1);
highlight.add(range2);
```